docs/docs/about/testimonials-quotes.mdx

```diff
@@ -67,4 +67,16 @@ If you would like to share your feedback with us, please [start a new discussion
   <div className='col testimonial-quote'>
     <p>I’ve been watching the project for quite some time, it’s very high quality, backed by a very rich group of contributors that actively expand the product. It has a set of unique capabilities that complement other existing tools, great samples, great documentation. I’ve always seen the CLI for Microsoft 365 project as an exemplar to how to create and maintain an awesome community with a great tool!</p>
   </div>
+</div>
+
+## NANDDEEP NACHAN
+
+<div className='row testimonial'>
+  <div className='col testimonial-image'>
+    <img src={require('!!url-loader!../images/testimonials-companies-logo/nanddeep-nachan.png').default} alt='Nanddeep Nachan' />
```

Comment:
Let's use the documented way of referencing pictures in Docusaurus. This will be important when we introduce versioning in our docs. Let's update the other pictures as well.

```suggestion
    <img src={require('@site/docs/images/testimonials-companies-logo/nanddeep-nachan.png').default} alt='Nanddeep Nachan' />
```

Reply:
last time I checked this totally did not work 😜. But I will recheck again

Reply:
but did you also try it for the .svg file?

Reply:
No, I haven't. I would expect it to work the same way.

Reply:
yes that is also what I expected but I already did that attempt and wrote it here 
https://github.com/pnp/cli-microsoft365/pull/6450#discussion_r1825291709
and at that time it worked for .png files but not for .svg file

Reply:
I'll try to look into it.

Reply:
@Adam-it, where is the SVG file actually? I don't see one on the testimonials page.

Reply:
aa sorry, There was one from Voitanos but I see this was updated by AC 🙂. Problem solved then